app/livechat/client/views/app/integrations/livechatIntegrationFacebook.html

```diff
@@ -9,8 +9,10 @@
 					{{else}}
 						<button class="button primary enable" {{enableButtonDisabled}}>{{_ "Enable"}}</button>
 						{{#unless hasToken}}
+							<div class="rc-input__wrapper">
 							<p>{{_ "You_have_to_set_an_API_token_first_in_order_to_use_the_integration"}}</p>
```

Comment:
Please, there are some missing tabs inside this div. Could you fix?

Reply:
@mariaeduardacunha I have made the changes please review